ophyd/status.py

```diff
@@ -536,6 +569,15 @@ def __str__(self):
 
     __repr__ = __str__
 
+    def _finished(self, success = True, **kwargs):
+        '''Inform the status object that it is done, and if it succeeded. If success is True
+        then also record telemetry about the move.
+        '''
+        
+        super()._finished(success = success, **kwargs)
+        if success:
+            self.pos.set.record(self)
```

Comment:
Is this missing an `est_time` accessor? Isn't `self.pos.set` the actual `set` method?

Reply:
yes fixed